documentation/concepts/00_accounts.md

```diff
@@ -104,13 +104,17 @@ Given global instantiated Aleo parameters and subroutines.
 1. Sample a 32 byte `seed` from random
     
 2. Construct private key components
-    - `sk_sig` = Blake2s(`seed`, 0)
-    - `sk_prf` = Blake2s(`seed`, 1)
-    - `r_pk` = Blake2s(`seed`, `counter`)
+    - `sk_sig` = BLAKE2s(`seed` | 0), where | denotes concatenation
+    - `sk_prf` = BLAKE2s(`seed` | 1)
+    - `r_pk` = BLAKE2s(`seed` | `counter`)
            
 3.`private_key` = (`seed`, `sk_sig`, `sk_prf`, `r_pk`)
 
-`counter` is a `u16` value that is iterated on until a valid `view_key` can be derived from `private_key`
+The 0 and 1 passed as second arguments to calculate `sk_sig` and `sk_prf` are encoded as unsigned 16-bit integers.
+`counter` is a `u16` value that is iterated on, starting from 2, until a valid `view_key` can be 
+derived from `private_key`. The endianness of the 2 bytes that form these u16s are little-endian.
```

Comment:
I would replace the sentence 'The endianness...' with 'Similarly to 0 and 1 used to calculate `sk_sig` and `sk_prf`, the unsigned 16-bit integer `counter` is turned into two bytes in little endian order before being concatenated to the right of the `seed`.